stestr/config_file.py

```diff
@@ -105,8 +105,8 @@ def get_run_command(self, test_ids=None, regexes=None,
 
         python = sys.executable if sys.platform == 'win32' else \
             '${PYTHON:-%s}' % (sys.executable)
-        command = "%s -m subunit.run discover -t" \
-                  " %s %s $LISTOPT $IDOPTION" % (python, top_dir, test_path)
+        command = '%s -m subunit.run discover -t "%s" "%s" ' \
```

Comment:
Instead of manually inserting quotes here could we just call `shlex.quote()` (although we'll have to use `shlex_quote` from `six.moves` because python 2 vs python 3) I think properly escaping the shell command we're building here is just a good thing to do.

Reply:
So I just tested this locally and using `shlex_quote()` doesn't have the desired behavior and it just breaks the execution of stestr. So leaving this as just quoted strings is fine.